srcpkgs/fail2ban/files/fail2ban/run

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
-mkdir -p /var/run/fail2ban
+mkdir -p /var/run/fail2ban /var/lib/fail2ban
```

Comment:
`/var/lib/fail2ban` is not on tmpfs unlike `/var/run/fail2ban`, so it can be created by the package on install time, not every time the service is enabled. Try creating this directory using `make_dirs` in the template and removing it from the service.

Reply:
Thanks, done.